cowprotocol/explorer
# Adding useQuery search URL params hook

# Summary
Closes #41 

Adding a hook that parses the search parameter query `tab` in the URL to differentiate **which tab view to display**, `orders` or `graph` nodes.
 


# To Test

1. Go to [/tx/HashId?tab=graph](https://pr49--gpui.review.gnosisdev.com/tx/0xc2eb04b549d68c770062591e2561c5b1534ea665c6179d33a779c52376c9161d?tab=graph)`
and you will see the graph instead of the order list.
